packages/react-docs/RELEASE-NOTES.md

```diff
@@ -4,6 +4,114 @@ section: get-started
+- **Label:** 
+  - Added support for React router ([#4934](https://github.com/patternfly/patternfly-react/pull/4934))
+- **Label group:** 
+  - Added label group main div for categories ([#4923](https://github.com/patternfly/patternfly-react/pull/4923))
+- **Modal:** 
+  - Added top align to modal ([#4871](https://github.com/patternfly/patternfly-react/pull/4871))
+- **Nav:**
+  - Added OUIA support into nav item ([#4769](https://github.com/patternfly/patternfly-react/pull/4769))
+- **Notification drawer:** 
+  - Added default variation to notification items ([#4792](https://github.com/patternfly/patternfly-react/pull/4792))
+  - Removed unused import ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Page:** 
+  - Replaced hasNoPadding property in the documentation example ([#4848](https://github.com/patternfly/patternfly-react/pull/4848))
+  - Fixed handleMainClick function signature ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Progress:** 
+  - Added warning variant ([#4834](https://github.com/patternfly/patternfly-react/pull/4834))
+- **Progress:** 
+  - Added truncation ([#4842](https://github.com/patternfly/patternfly-react/pull/4842))
+- **Select:** 
+  - Added support for divider in select ([#4772](https://github.com/patternfly/patternfly-react/pull/4772))
+  - Cleaned up some typeahead behaviors ([#4864](https://github.com/patternfly/patternfly-react/pull/4864))
+  - Fixed grouped Select generating invalid HTML ([#4919](https://github.com/patternfly/patternfly-react/pull/4919))
+  - Added support to pass chip group props on typeaheadmulti variant ([#4855](https://github.com/patternfly/patternfly-react/pull/4855))
+  - Updated to return the string value for select chips ([#4915](https://github.com/patternfly/patternfly-react/pull/4915))
+  - Passed aria-label / aria-labelledby to the `<ul role="listbox">` to prevent axe issues ([#4931](https://github.com/patternfly/patternfly-react/pull/4931))
+  - Updated to remove the clear button button when onClear function is not specified for typeahead ([#4887](https://github.com/patternfly/patternfly-react/pull/4887))
+- **Tabs:** 
+  - Added support for light scheme variant in Tabs && TabContent ([#4832](https://github.com/patternfly/patternfly-react/pull/4832))
+  - Conditionally render aria-controls when mountOnEnter ([#4796](https://github.com/patternfly/patternfly-react/pull/4796))
+- **Tooltip** 
+  - Updated to export TooltipProps as a type to help babel ([#4854](https://github.com/patternfly/patternfly-react/pull/4854))
+
+### Catalog view extension:** 
+- Added back catalogue items ([#4940](https://github.com/patternfly/patternfly-react/pull/4940))
+
+### Charts
+- Replaced or used exact lodash imports ([#4830](https://github.com/patternfly/patternfly-react/pull/4830))
+
+### Icons
+- Used lying cast ([#4935](https://github.com/patternfly/patternfly-react/pull/4935))
```

Comment:
Do we want a more descriptive summary here, such as "Revert support for svg alt tags"?